mrp_bom_version/README.rst

```diff
@@ -82,4 +85,4 @@ Credits
 Contributors
 ------------
 
-** to do **
+* Eficent, www.eficent.com
```

Comment:
If you take a look to [`__openerp__.py` file](https://github.com/odoomrp/odoomrp-wip/blob/8.0/mrp_bom_version/__openerp__.py#L29) you will see that there are some contributors, as listed below:
```
* Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
* Ana Juaristi <anajuaristi@avanzosc.es>
* Alfredo de la Fuente <alfredodelafuente@avanzosc.es>
```
So you can add them in the README.rst file.